### PR TITLE
show teacher name info

### PR DESCRIPTION
**Pull request by smallst**
see server changes: https://github.com/codecombat/codecombat-server/pull/848
fix ENG-262
image
image
image

### CODE REVIEW

File: app/views/outcomes-report/OutcomesReportResultComponent.vue [hunk 1]
```diff
@@ -236,6 +236,24 @@ export default Vue.extend({
...
+        return `<a href="/outcomes-report/teacher/${tea._id}">${tea.firstName}</a>`
+      }
+      const info = $.i18n.t('outcomes.top_teacher_info', {
+        A: formatTeacher(this.org.topTeachers[0]),
+        B: formatTeacher(this.org.topTeachers[1]),
+        n: this.org.nces.teachers - 2
+      })
+      // $.i18.t encode the html
+      const txt = document.createElement('textarea')
+      txt.innerHTML = info
```
Comment (mrfinch) [inline app/views/outcomes-report/OutcomesReportResultComponent.vue:255]: do we need to do it like this? Why not pass values as variables to vue template instead? Could also just create a component which accepts these parameters

File: app/views/outcomes-report/OutcomesReportResultComponent.vue [hunk 2]
```diff
@@ -236,6 +236,24 @@ export default Vue.extend({
...
+    },
+
+    topTeacherInfo () {
+      const formatTeacher = (tea) => {
+        if (!tea) {
+          return ''
+        }
+        return `<a href="/outcomes-report/teacher/${tea._id}">${tea.firstName}</a>`
+      }
+      const info = $.i18n.t('outcomes.top_teacher_info', {
```
[the remark this answers is not in the document]
Comment (smallst) [reply app/views/outcomes-report/OutcomesReportResultComponent.vue:248]: no teachers passed is not a problem. it just show empty `<a>` show nothing happens

File: app/views/outcomes-report/OutcomesReportResultComponent.vue [hunk 3]
```diff
@@ -236,6 +236,24 @@ export default Vue.extend({
...
+        A: formatTeacher(this.org.topTeachers[0]),
+        B: formatTeacher(this.org.topTeachers[1]),
+        n: this.org.nces.teachers - 2
```
Comment (mrfinch) [inline app/views/outcomes-report/OutcomesReportResultComponent.vue:251]: need to use teacherCount

File: app/views/outcomes-report/OutcomesReportResultComponent.vue (hunk 1 above)
Comment (smallst) [reply app/views/outcomes-report/OutcomesReportResultComponent.vue:255]: ya we can use component too. but component may complex than a single document element?

**Comment by smallst** [comment]
if we want to update one line string to component we can add new tasks to do that. let's merge this one first

**Pull request merged**